### PR TITLE
Add kratos rock release

### PR DESCRIPTION
**Pull request by nsklikas**
Add files for the [Kratos ROCK](github.com/canonical/kratos-rock/)

Didn't add any documentation or contacts, hope that's ok.

### CODE REVIEW

**Comment by cjdcordeiro** [comment]
Thanks for this contribution. This seems to be a new ROCK and you're asking for it to be released into DH and ECR, so before even starting the review, I suggest you go through the Contributor's Contract (find the link from last week's workshop) and reach out to @valentincanonical about whether this should be under the "ubuntu" namespace or not

File: oci/kratos/builds.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: 1
+
+images:
+  - source: "canonical/kratos-rock"
+    commit: 07ba17388852851a2b731d9b01f55f95271572d6
+    directory: .
+    release-to:
+      risks:
+        - edge
```
Comment (valentincanonical) [inline oci/kratos/builds.yaml:9]:
```suggestion
```

**Review by valentincanonical — CHANGES_REQUESTED** [review]
As discussed, this image might eventually make it to the Ubuntu namespace as part of Canonical's images portfolio but this will happen later when there's better testing, documentation, and the team is ready to commit to supporting these images. 

For now, I suggest removing the release-to / releases options, so that the OCI factory will still build these Rocks but not publish them anywhere beyond the staging GHCR area.

**Comment by cjdcordeiro** [comment]
Hi again! I'm just reviving this thread to know we it should be resumed or closed ;)

(FYI @nsklikas )

**Pull request closed without merging**